### PR TITLE
added a new unlaunch bg

### PR DESCRIPTION
**Pull request by Emran54320**
New unlaunch bg added: Call Of Duty : Modern Warfare 2

### CODE REVIEW

File: _nds/TWiLightMenu/unlaunch/backgrounds/_meta.json [hunk 1]
```diff
@@ -63,6 +63,11 @@
 		"author": "SNBeast",
 		"categories": ["vista","windows"]
 	},
+	"codmw2": {
+		"title": "Call Of Duty : Modern Warfare 2",
+		"author": "Emran_Ahmed",
+		"categories": ["call of duty","games"]
```
Comment (Epicpkmn11) [inline _nds/TWiLightMenu/unlaunch/backgrounds/_meta.json:69]: Oh, can you make the category not have spaces? I think that'll mess with it

```suggestion
		"categories": ["call-of-duty","games"]
```
or
```suggestion
		"categories": ["cod","games"]
```

Comment (Emran54320) [reply _nds/TWiLightMenu/unlaunch/backgrounds/_meta.json:69]: so i need to do another pr?

Comment (Epicpkmn11) [reply _nds/TWiLightMenu/unlaunch/backgrounds/_meta.json:69]: No, just make another commit to this branch. You can also just do the `Commit suggestion` on either of the suggestions I did in that comment.

**Comment by Emran54320** [comment]
okay

**Pull request merged**